components/servo.rst

```diff
@@ -55,6 +55,9 @@ Advanced Options:
 - **restore** (*Optional*, boolean): Whether to restore the state of the servo motor at startup.
   This is useful if you have an absolute servo motor and it goes back to its 0 position at startup.
   Defaults to ``false``.
+- **auto_detach_time** (*Optional*, ms): The time after reaching the target value when the servo will be detached.
+- **transition_length** (*Optional*, ms): The needed for a full movement (-1.0 to 1.0).
```

Comment:
```suggestion
- **transition_length** (*Optional*, :ref:`config-time`): The time needed for a full movement (-1.0 to 1.0). This will effectively limit the speed of the servo, the larger the value, the slowest the servo will move. Defaults to `` 0s``
```